### PR TITLE
Topic recognition-s4549518-Zhipei Tao-Improved_Unet on ISIC dataset

### PR DESCRIPTION
**Pull request by TTTZP**

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific
- Did not follow required project structure. Your directory should be in `recognition` folder
- Code valid for Easy/Normal Difficulty
- Only training and validation sets, no test set

Other Feedback
- Please group the `.png` files and put them in a sub-directory
- Please follow python file naming convention e.g. `lorem_ipsum_123_abc.py`

**Comment by rexsw** [comment]
Code looks good for easy difficulty

Your pull request should include a description of what is in it and why it should be merged

Try not to have code just floating around like line 10 of improved_unet_model.py, that code should be in a function.

Some files lack meaningful inline comments, these are needed for any complex project 

Tensorflow has functions for reading and resizing images so there is no need to use cv2 for this. It is always good to minimize the number of libraries your project needs.

Code in files such as evaluate_model should be in a main function 

Keep in mind paths such as 'Downloads/ISIC2018_Task1-2_Training_Input_x2' might not work for users, these either need to be set dynamically or the readme needs to make it clear that the user needs to edit these paths

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Partially Pull Request (Working Algorithm Delivered on Time but in Incorrect Branch) 
**Feedback required. Move your folder into the recognition folder, move images into sub directory, ensure no conflicts.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by shakes76** [comment]
Thanks, but the readme file for the recognition folder is still being overridden, needs to be reverted before I can merge.
